R/integration.R

```diff
@@ -99,8 +99,8 @@ runHarm = function(sobj, batch) {
 	sobj <- ScaleData(sobj)
 	sobj <- RunPCA(sobj, features=rownames(sobj@assays$RNA))
 	sobj <- RunHarmony(sobj, batch)
-	sobj@reductions['X_emb'] <- sobj@reductions$harmony
-	#harmonyEmb <- HarmonyMatrix(pca, method, batch, do_pca=F)
+	sobj[['X_emb']] <- sobj[['harmony']]
+    #harmonyEmb <- HarmonyMatrix(pca, method, batch, do_pca=F)
```

Comment:
could we also remove the comment while we're at it?

Reply:
This comment was already there. I'm happy to remove it but I think there are a few others in this file as well.

Reply:
yeah, i saw... community service: clean up after others :)